gensim/models/poincare.py

```diff
@@ -73,22 +74,32 @@ class PoincareModel(utils.SaveLoad):
     and :meth:`~gensim.models.poincare.PoincareModel.load` methods, or stored/loaded in the word2vec format
     via `model.kv.save_word2vec_format` and :meth:`~gensim.models.poincare.PoincareKeyedVectors.load_word2vec_format`.
 
-    Note that training cannot be resumed from a model loaded via `load_word2vec_format`, if you wish to train further,
+    Notes
```

Comment:
What's about examples here?

Reply:
We have example for each method in the `PoincareKeyedVectors` class.